applications/GeoMechanicsApplication/tests/moving_column_with_fixed_pressure_above_phreatic_line/with_move_mesh/ProjectParameters.json

```
@@ -0,0 +1,166 @@
+                "GiDPostMode": "GiD_PostAscii",
+                "MultiFileFlag": "SingleFile"
+              },
+              "file_label": "step",
+              "output_control_type": "step",
+              "output_interval": 1,
+              "body_output": true,
+              "node_output": false,
+              "skin_output": false,
+              "plane_output": [],
+              "nodal_results": ["WATER_PRESSURE", "TOTAL_DISPLACEMENT"],
+              "gauss_point_results": []
+            },
+            "point_data_configuration": []
+          }
+        }
+      }
+    ]
+  },
+  "processes": {
+    "constraints_process_list": [
+      {
+        "python_module": "apply_scalar_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name": "ApplyScalarConstraintTableProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.Top",
+          "variable_name": "WATER_PRESSURE",
+          "is_fixed": true,
+          "table": 0,
+          "fluid_pressure_type": "Phreatic_Multi_Line",
+          "gravity_direction": 1,
+          "out_of_plane_direction": 2,
+          "x_coordinates": [0.0, 1.0],
+          "y_coordinates": [-2.0, -2.0],
+          "z_coordinates": [0.0, 0.0],
+          "specific_weight": 9810.0
+        }
+      },
+       {
+        "python_module": "apply_scalar_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name": "ApplyScalarConstraintTableProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.Bottom",
+          "variable_name": "WATER_PRESSURE",
+          "is_fixed": true,
+          "table": 0,
+          "fluid_pressure_type": "Phreatic_Multi_Line",
+          "gravity_direction": 1,
+          "out_of_plane_direction": 2,
+          "x_coordinates": [0.0, 1.0],
+          "y_coordinates": [-2.0, -2.0],
+          "z_coordinates": [0.0, 0.0],
+          "specific_weight": 9810.0
+        }
+      }
+    ],
+    "loads_process_list": [
+      {
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name": "ApplyVectorConstraintTableProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.Soil",
+          "variable_name": "VOLUME_ACCELERATION",
+          "active": [false, true, false],
+          "value": [0.0, -9.81, 0.0],
+          "table": [0, 0, 0]
+        }
+      },{
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name": "ApplyVectorConstraintTableProcess",
+        "Parameters": {
+          "model_part_name": "PorousDomain.Soil",
+          "variable_name": "DISPLACEMENT",
+          "active": [true, true, true],
+          "is_fixed": [true, true, true],
```

I'm just wondering: since this is a plane strain model, can't we make the z components inactive?
```suggestion
          "active": [true, true, false],
          "is_fixed": [true, true, false],
```